app/src/main/java/com/github/multimatum_team/multimatum/activity/MainActivity.kt

```diff
@@ -53,7 +54,6 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
 
         Firebase.initialize(this)
-        pdfRepository = FirebasePdfRepository(FirebaseStorage.getInstance())
```

Comment:
Please use dependency injection
```kt
@Inject
lateinit var pdfRepository: PdfRepository
```
with the repository provider in `DependencyProvider.kt`